pkg/stanza/adapter/storage_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+
```

Comment:
Your editor conflicts with the CI linters:
```
stanza/adapter/storage_test.go:11: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
```

You can just delete this line

```suggestion
```

Reply:
I don't know how this happened.. might need to change some setting in GoLand